src/components/ModalPurchaseHbar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <Modal
+        :is-open="isOpen"
+        hide-decoration
+        @change="this.$listeners.change"
+    >
+        <iframe
+            id="carbonFiberIFrameExample"
+            title="Carbon Fiber iframe Example"
+            width="450"
+            height="800"
+            allow="fullscreen"
+            :src="url"
+        />
+    </Modal>
+</template>
+
+<script lang="ts">
+import Modal from "../components/Modal.vue";
+import {
+    createComponent,
+    PropType,
+    computed
+} from "@vue/composition-api";
+import { store } from "../store";
+
+export default createComponent({
+    components: { Modal },
+    model: {
+        prop: "isOpen",
+        event: "change"
+    },
+    props: { isOpen: (Boolean as unknown) as PropType<boolean> },
+    setup() {
+        const accId = store.state.wallet.session!.account;
```

Comment:
`getters.GET_WALLET()`

or implement 

`getters.CURRENT_USER()`

or something like that